Plugin/Source/Processors/Timing_Effects/OHProcess.h

```diff
@@ -2,6 +2,7 @@
 #define OHPROCESS_H_INCLUDED
 
 #include <JuceHeader.h>
+#include <chowdsp_sources/Other/chowdsp_Noise.h>
```

Comment:
Ah so here, I think we need to add `chowdsp_sources` to the modules that the plugin is linking to. This should be done [here](https://github.com/jatinchowdhury18/AnalogTapeModel/blob/master/Plugin/modules/CMakeLists.txt#L28)

Reply:
Ah right thanks